LayoutTests/media/media-vp9-sar-webm.html

```diff
@@ -0,0 +1,23 @@
+<html>
+<head>
+<title>webm file with vp9 with SAR 16:9</title>
+<script src="../media/video-test.js"></script>
+<script>
+    async function runTest()
+    {
+        findMediaElement();
+        run('video.src = "content/test-par-16-9.webm"');
+        await waitFor(video, 'loadedmetadata');
+        testExpected('video.videoWidth', 320);
```

Comment:
I don't think this test is sufficient to check that we properly render the AR.
videoWidth/videoHeight would have been correct before, they would just have been rendered incorrectly.

A reftest may be required here, or checking the width/height as well (not verified)